assets/javascripts/discourse/services/chat.js

```diff
@@ -851,7 +852,15 @@ export default class Chat extends Service {
       data.data = JSON.stringify(draft);
     }
 
-    ajax("/chat/drafts", { type: "POST", data });
+    ajax("/chat/drafts", { type: "POST", data, ignoreUnsent: false })
+      .then(() => {
+        this.set("isNetworkUnreliable", false);
+      })
+      .catch((error) => {
+        if (!error.jqXHR?.responseJSON?.errors?.length) {
+          this.set("isNetworkUnreliable", true);
```

Comment:
Do we make a recurring polling call to `/chat/drafts`? I am wondering whether we need a way to periodically (every 30 seconds or minute) ensure that the network is still unreliable, otherwise we could be showing the icon to a lurker for a long time (i.e. until they next try to send).

Reply:
Yes that's why I  noted early that I might add this network stuff in more ajax calls

Reply:
Honestlly I'm not a big fan of the 30 seconds network poll, but what we could do, is reseting network state after 30s

that way we ensure that if no action are taken, we don't stay in this state forever.

Reply:
Oh yeah, that's good! Reset after 30/60s makes sense, no need to stay in "alert mode" forever.